### ISSUE TITLE
Exception encountered when calling layer "vgg16" (type Functional).

### ISSUE THREAD

**Issue by Jennyfer5364** [MathiasGruber/PConv-Keras#78]
Exception encountered when calling layer "vgg16" (type Functional).

pop from empty list

Call arguments received:
  • inputs=tf.Tensor(shape=(None, 512, 512, 3), dtype=float32)
  • training=None
  • mask=None

anyone help me to solve this why this happening

**Comment by dkncus** [comment]
I am having the same issue. Is there a known workaround/fix?

**Comment by dkncus** [comment]
Further notes - I am attempting to load the pretrained model provided in the repo. It seems this is an issue in the 'build_vgg' function in 'pconv_model.py'. The specific line being called causing this error is model= Model(inputs=img, outputs=vgg(processed)). I've managed to isolate the source of this error specifically to the call "vgg(processed)".

**Comment by Draco18s** [comment]
I am also seeing this issue.

**Comment by dkncus** [comment]
I was never able to resolve it fully. What are you trying to do with it?

**Comment by Draco18s** [comment]
I was hoping to be able to fine-tune the model on specific content. But if I can't, oh well.

**Comment by KireaSamuel** [comment]
i also have this issue

**Comment by DenisUllmann** [comment]
I got the same problem.
It seams to be a matter of how to define the outputs of the vgg model with recent versions of TF and Keras.
Instead of:
`vgg.outputs = [vgg.layers[i].output for i in vgg_layers]`
You can try:
`vgg = Model(vgg.inputs, [vgg.layers[i].output for i in vgg_layers])`
